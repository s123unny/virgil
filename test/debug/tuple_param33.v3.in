b ../core/tuple_param33.v3 4
b ../core/tuple_param33.v3 7
b ../core/tuple_param33.v3 10
run
bt
info l
c
bt
info l
c
bt
info l
c
q
