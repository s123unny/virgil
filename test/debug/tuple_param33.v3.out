breakpoint|../core/tuple_param33.v3 4|0
breakpoint|../core/tuple_param33.v3 7|1
breakpoint|../core/tuple_param33.v3 10|2
stopOnBreakpoint
bt|tuple_param33.main|../core/tuple_param33.v3|4
variable|1|a|0|int|false
stopOnBreakpoint
bt|tuple_param33.apply|../core/tuple_param33.v3|7
bt|tuple_param33.main|../core/tuple_param33.v3|4
variable|1|f|<null>:tuple_param33.m|T -> int|false
variable|2|p|(<null>, 1, 2, 3, 4, 5, 6, 7, 8, 9)|T|false
stopOnBreakpoint
bt|tuple_param33.m|../core/tuple_param33.v3|10
bt|tuple_param33.apply|../core/tuple_param33.v3|7
bt|tuple_param33.main|../core/tuple_param33.v3|4
variable|0|this|tuple_param33|tuple_param33|false
variable|1|a|0|int|false
variable|2|b|1|int|false
variable|3|c|2|int|false
variable|4|d|3|int|false
variable|5|e|4|int|false
variable|6|f|5|int|false
variable|7|g|6|int|false
variable|8|h|7|int|false
variable|9|i|8|int|false
variable|10|j|9|int|false
end
result|45
